en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md

```diff
@@ -0,0 +1,262 @@
+# Amazon Lambda Connector Example
+
+AWS Lambda is a compute service that lets you run code without provisioning or managing servers. AWS Lambda executes your code only when needed and scales automatically, from a few requests per day to thousands per second. 
+The following operations allow you to work with functions in Amazon Lambda.
```

Comment:
Seems irrelevant since we are not listing operations here. Let's remove this line,

Reply:
Removed.